text/0000-mixed-shadow-mode.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Mixed Shadow Mode
+status: DRAFTED
+created_at: 2021-05-07
+updated_at: YYYY-MM-DD
+pr: https://github.com/salesforce/lwc-rfcs/pull/49
+---
+
+# Mixed Shadow Mode
+
+## Introduction
+
+LWC offers a collection of polyfills that implement Shadow DOM features. These polyfills are
+collectively referred to as "synthetic Shadow DOM" and are published to npm under the
+`@lwc/synthetic-shadow` package. Synthetic shadow DOM provides a consistent experience for LWC web
+components across the Salesforce-supported browser matrix.
+
+Native Shadow DOM support has greatly increased since the start of the LWC project but applications
+continue to rely on the synthetic shadow polyfills to maintain backwards-compatibility. The removal
+of these polyfills can result in broken experiences due to dependencies on certain "escape hatches"
+that are not possible in a native Shadow DOM environment (e.g., instrumentation, styling, etc, which
+are discussed below).
+
+Applications that wish to use LWC components in a native shadow context simply need to omit the
+inclusion of the `@lwc/synthetic-shadow` polyfill; however, this is not a realistic option for
+large applications that cannot afford a rewrite.
+
+This proposal opens up an incremental migration path for applications moving to native Shadow DOM by
+allowing the usage of both native and synthetic Shadow DOM in the same document.
+
+## Detailed design
+
+Components will opt-in to this new feature and the default mode will be synthetic mode. From the

```

Comment:
this first sentence is confusing in the sense that it looks like the component is choosing synthatic mode by default, but that's only true if the app chooses to load synthetic shadow, which platform always does.